public/app/features/explore/QueryLibrary/QueryTemplatesTable/ActionsCell.tsx

```diff
@@ -35,7 +36,11 @@ function ActionsCell({ queryTemplate, rootDatasourceUid, queryUid }: ActionsCell
 
   const onDeleteQuery = (queryUid: string) => {
     const performDelete = (queryUid: string) => {
-      deleteQueryTemplate({ uid: queryUid });
+      deleteQueryTemplate({
+        name: queryUid,
+        namespace: getK8sNamespace(),
+        ioK8SApimachineryPkgApisMetaV1DeleteOptions: {},
```

Comment:
this should be optional in the api spec 😬

Reply:
ah, this is optional in the spec. looks like a bug with `rtk-query-codegen-openapi` client generation 😭